src/stun_listener.erl

```diff
@@ -159,7 +161,12 @@ start_listener(IP, ClientPort, {MinPort, MaxPort}, Transport, Opts, Owner)
     end;
 start_listener(IP, ClientPort, {MinPort, MaxPort}, udp, Opts, Owner) ->
     OpenFun =
-        fun(Port) -> gen_udp:open(Port, [binary, {ip, IP}, {active, false}, {reuseaddr, true}])
+        fun(Port) -> gen_udp:open(Port, [binary, 
+            {ip, IP}, 
+            {active, false},
+            {recbuf, ?UDP_RECBUF},
+            {read_packets, ?UDP_READ_PACKETS}, 
```

Comment:
https://www.erlang.org/doc/man/gen_udp.html
"The option [read_packets](https://www.erlang.org/doc/man/inet.html#option-read_packets) is currently ignored."